topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -0,0 +1,444 @@
+
+> <hands-on-title> Plot for gene expression </hands-on-title>
+>
+> 1. {% tool [Scanpy PlotEmbed](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_plot_embed/scanpy_plot_embed/1.8.1+galaxy9) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in AnnData/Loom format"*: `output_h5ad` (output of **Scanpy RunFDG** {% icon tool %})
+>    - *"name of the embedding to plot"*: `draw_graph_fa`
+>    - *"color by attributes, comma separated texts"*: `ENSMUSG00000023274,ENSMUSG00000053977`
+>    - *"Use raw attributes if present"*: `No`
+>    - *"Location of legend"*: `On data`
+>
+>    > <comment-title> Gene Symbols </comment-title>
+>    >
+>    > We're using the EnsemblIDs during this tutorial, as discussed above. If you like, you could change the names of these plots to the gene symbols by filling in the optional Figure title field with `Cd4,Cd8`. Make sure that the order of your figure titles matches the order of the EnsemblIDs in the colour by field. ENSMUSG00000023274 is Cd4 and ENSMUSG00000053977 is Cd8.
+>    {: .comment}
+> 
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. Does the expression pattern of these genes tell us anything about our cells?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. It's clear that both Cd4 and Cd8 are being expressed mainly in the later stages of T-cell development, as they head towards the DP-L cluster - although Cd4 expression is a bit more widespread. This is what we would expect to see in genes that are associated with mature T-cells. You might also be able to spot some differences in the expression of the two genes in our mature T-cell group, but there doesn't seem to be a very clear division between the cells that express Cd4 and those that express Cd8. This is a bit disappointing, as we know that there are two types of mature T-cells, which each express a different gene. 
+> >
+> > ![Cells in the DP L cluster are coloured as having the highest expression levels of both Cd4 and Cd8 while most cells in the M4 cluster already show higher expression of Cd8](../../images/scrna-case_trajectories/TrajectoriesFDGExpression.png "FDG plot showing expression of Cd4 and Cd8")
+> {: .solution}
+>
+{: .question}
+
+## Diffusion pseudotime
+
+Now that we have a reasonable FDG plot for our cells, based on the diffusion map (if used) and PAGA plot, we can place our cells into pseudotime. Pseudotime lets us imagine that instead of looking at a sample of cells taken at a single timepoint, we are looking at cells moving through time. Our sample included cells at different stages of their development, but we can use pseudotime to think of these as different timepoints in the journey of individual cells. 
+
+We know that our cells are initialising at DN. We can feed that information into our algorithms by naming DN as the root cell type to then calculate a trajectory starting from these cells. 
```

Comment:
maybe a quick reminder of what DN means as a cell type, (as before, only if it's not obvious for people working with this data)